### ISSUE TITLE
CFQ001 counts too many lines

### ISSUE THREAD

**Issue by alexis974** [best-doctor/flake8-functions#9]
**Bug description:**
CFQ001 takes into account empty lines and comment lines when counting number of lines inside a function.

**To Reproduce:**
1. Create a function that contains some empty lines and/or comments lines
2. Run flake8 with the `--max-function-length=N` where N is the number of line of your function (without counting the empty line and comment line)

**Expected behavior:**
CFQ001 should only count lines of actual code. Taking into account empty lines and comment lines may lead to developers removing some comments or empty lines to fix CFQ001 linting error. Thus, making the code less readable.

**Desktop:**
 - OS:Manjaro Linux x86_64 
 - Python : 3.9.7
 - Flake8: 4.0.1
 - Flake8-functions : 0.0.6

**Comment by jeffnyman** [comment]
Agreed and seconding this as a concern. I'm finding exactly what was described in the initial report: developers started removing comments -- and necessary and helpful comments -- from functions just to satisfy this lint check.

Length from a linting perspective should only be taking into account executable lines of code. That said, I could see a desire by some to also check for all lines, including non-executable ones. In that case, maybe there should be a configuration option? Something like `count-only-executable-lines` or `include-non-executable-lines` or whatever makes sense.

**Comment by alexis974** [comment]
Great idea to add a configuration option for this. I think the default behavior should be to omit non-executable lines. 

I would be happy to implement this feature enhancement. I just need the approval of a maintainer.

Also, [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) says the following thing :
> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list [...] or it may be lined up under the first character of the line that starts the multiline construct:

How should we consider lines with just a closing brace/bracket/parenthesis ? For example, should we count 3 or 4 lines for the following code :
```Python
my_dict = {
    “one”: 1,
    “two”: 2,
    }
```

**Comment by justkrys** [comment]
Agreed +1 on fixing this bug.  Comments should not be counted and I am being bitten by this right now. :(
Thanks for the great tool though!
